script/Ciel/Day 10/10_01_CIEL10_4_1.txt

```diff
@@ -126,57 +141,62 @@ Or even if she drinks herself silly after work."
 [sha:0d6e38998772153e7dccc84af60f8c72c9d90912]{
 -- Page 178, Offset 27546.
 -- それは不仲ではない、というより、なんの交友もない、というのではないだろうか？
-It seems less like a matter of being on bad terms, and more like there isn't any camaraderie between them, doesn't it?
+This doesn't seem like an issue of them not being on great terms.
+ It's more like they have absolutely no attachment to each other at all.
 }
 [sha:b5680e8541edf535c304d090e86b825fa94f0e04]{
 -- Page 178, Offset 27547.
 -- ……シエル先輩、友達作り下手なのかな……。
-...Is Ciel-senpai bad at making friends?
+Is Ciel-senpai... bad at making friends...?
 }
 [sha:9cec6cbe9499dac97b87061ba79597448ab6d942]{
 -- Page 179, Offset 27548.
 -- 「じゃあ吸血鬼を退治する時以外、会ったりはしていない、と……それもそうか。
-"Meaning that except when exterminating vampires, you don't meet at all... Makes sense.
+"So the two of you don't meet up unless you're exterminating vampires...
+ That makes sense I suppose.
 }
 [sha:8346b7996dfc7623cbc4e9e38fa18a822c38cd03]{
 -- Page 179, Offset 27549.
 -- 歳も離れているし、趣味も違いそうだし、休日一緒に過ごしている二人を想像できないし」
-She's older than you and you don't have the same interests, I can't imagine the two of you being together on a free day."
+She's older than you and I get the feeling you don't really share any interests,
+ so I guess I can't really picture you hanging out in your free time."
 }
 [sha:f29ce5a86bb3af7eb77893800f3526081640f6f7]{
 -- Page 180, Offset 27550.
 -- 「…………歳が離れている…………。
-"She's... older than me...
+"She's... older than me...?
 }
 [sha:c799fe205cee2d844321c889459897b750e7703e]{
 -- Page 180, Offset 27551.
 -- 参考までに質問させてください。わたしくらいの子が、ノエル先生と一緒にいるのは不自然ですか？」
-Allow me to ask you a question just for information. Does it look strange that Noel-sensei partners with someone like me?"
+Just out of interest――
+does it seem strange to you at all that someone my age is partnered with Noel-sensei?"
 }
 [sha:f9f967a2057a8f3158de7ed62cf417f02f73ba64]{
 -- Page 181, Offset 27552.
 -- 「まあ、教師と生徒の関係ですから。
-"Well, one is the teacher and the other is the student.
+"I mean, you're student and teacher, right?
 }
 [sha:670759126a4409620b45a900284346dd09a903fa]{
 -- Page 181, Offset 27553.
 -- ノエル先生、教師としては若いといっても25歳だし」
-Noel-sensei, though young for a teacher, is twenty-five years old."
+Noel-sensei's a bit young for a teacher, but she's still twenty-five."
 }
 [sha:283ee2efbd144ad7cf0517dc3a1fc2287fcb3c37]{
 -- Page 181, Offset 27554.
 -- 「ま゛？」
-"Wha...?"
+"Wha...?!"
```

Comment:
I think em-dash is more appropriate.

Though to be fair I don't think we've quite established a standard on interruptions.